README.md

```diff
@@ -542,3 +548,60 @@ And run the tests from the root of the source code:
 ```text
 bundle exec rake parallel_spec
 ```
+
+See also `.travis.yml` for information on running the acceptance and other tests.
```

Comment:
there is no travis.yml anymore.

Reply:
Removed and replaced with a mention of the Github Actions